src/Sulu/Bundle/AudienceTargetingBundle/Resources/public/js/services/target-groups/manager.js

```diff
@@ -0,0 +1,37 @@
+define(['underscore', 'jquery', 'services/husky/util'], function(_, $, Util) {
+
+    'use strict';
+
+    var instance = null,
+
+        getInstance = function() {
+            if (instance === null) {
+                instance = new TargetGroupManager();
+            }
+            return instance;
+        },
+
+        url = _.template('/admin/api/target-groups<% if (typeof id !== "undefined") { %>/<%= id %><% } %>');
+
+    /** @constructor **/
+    function TargetGroupManager() {
+    }
+
+    TargetGroupManager.prototype = {
```

Comment:
Just return the object you are assigning here. All this `getInstance` and constructor stuff here is only unnecessary overhead.